__tests__/app.test.js

```diff
@@ -41,3 +44,54 @@ describe("GET API", () => {
       });
   });
 });
+
+describe("GET /api/article/:article_id", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should return an object with the correct article with correct keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body.article).toBe("object");
+        expect(typeof res.body.article[0].title).toBe("string");
```

Comment:
if you're getting a specific article we don't care about the types of all the properties, you should be making sure the object looks how you expect and it is article id 1, you know what the title and votes are etc. so make your test stricter by asserting against them. Try using .objectContaining or .toMatchObject here